app/controllers/users_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
-    before_action :authorize, only: [:show]
+
     def create
```

Comment:
There is a validation method in the application controller that caters for all validations. Use the appropriate create and update

Reply:
I have seen it